packages/ApiTests/src/viewers/filters.ts

```diff
@@ -0,0 +1,74 @@
+import * as ui from 'datagrok-api/ui';
+import * as grok from 'datagrok-api/grok';
+import * as DG from 'datagrok-api/dg';
+
+import {
```

Comment:
some unnecessary imports

Reply:
I import namespaces grok, DG and ui in all modules.
https://datagrok.slack.com/archives/C02AJ2FT2F5/p1679389910994899